c2_utils/src/mfx_defs.cpp

```diff
@@ -81,6 +81,7 @@ mfxStatus MFXLoadSurfaceSW(uint8_t *data, uint32_t stride, const mfxFrameInfo& i
     if (!MFX_C2_IS_COPY_NEEDED(srf->Data.MemType, input_info, srf->Info)) {
         srf->Data.Y  = data;
         srf->Data.UV = data + nOWidth * nOHeight;
+        srf->Data.V = srf->Data.UV + 1;
```

Comment:
Is it better to make changes like below?
srf->Data.U = data + nOWidth * nOHeight;
srf->Data.V = srf->Data.U + 1;